_pages/main.md

```diff
@@ -5,6 +5,12 @@ type: single
 permalink: /
 ---
 
+
+## Join the 'Insects' breakout session at this year's G10K-VGP EBP 2019 Meeting! This year's meeting is a joint effort between the Vertebrate Genomes Project (VGP) and the Earth BioGenome Project (EBP). The discussion-based insects breakout session is scheduled for 9am -12 pm Eastern, Wednesday, August 28th. If you are not coming to the meeting, but are interested in shaping the discussion about insects or arthropods in the Earth BioGenome Project, we encourage you to attend the session remotely.
+* [Read the Insects breakout session agenda](ebp_insects_agenda) 
```

Comment:
This gave me a 404 error when I tested it on my local.  
ebp_insects_agenda.md is the page.  I